lib/screen/boarding_screen.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/material.dart';
+import 'package:sochem/screen/slide.dart';
+
+class BoardingPage extends StatefulWidget {
+  @override
+  _BoardingScreenState createState() => _BoardingScreenState();
+}
+
+class _BoardingScreenState extends State<BoardingPage> {
+  int _currentPage = 0;
+  List<Slide> _slides = [];
+  PageController _pageController = PageController();
+
+  @override
+  void initState() {
+    _currentPage = 0;
+    //put slide content here
+    _slides = [
+      Slide("assets/img1.png", "Replace this text with heading of first slide",
+          "Replace this text with content of first slide"),
+      Slide("assets/img2.png", "Replace this text with heading of second slide",
+          "Replace this text with content of second slide"),
+      Slide("assets/img3.png", "Replace this text with heading of third slide",
+          "Replace this text with content of third slide"),
+    ];
+    _pageController = PageController(initialPage: _currentPage);
+    super.initState();
+  }
+
+  List<Widget> _buildSlides() {
+    return _slides.map(_buildSlide).toList();
+  }
+
+  Widget _buildSlide(Slide slide) {
+    return Column(
+      children: <Widget>[
+        Expanded(
+          child: Container(
+            margin: EdgeInsets.all(32),
+            child: Image.asset(
+              slide.image,
+              fit: BoxFit.contain,
+            ),
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: 70),
+          child: Text(
+            slide.heading,
+            textAlign: TextAlign.left,
+            style: TextStyle(
+              fontSize: 24,
+              fontWeight: FontWeight.w900,
+            ),
+          ),
+        ),
+        Padding(
+          padding: EdgeInsets.symmetric(horizontal: 70),
+          child: Text(
+            slide.content,
+            textAlign: TextAlign.left,
+            style: TextStyle(
+              fontSize: 18,
+              fontWeight: FontWeight.w700,
+            ),
+          ),
+        ),
+        SizedBox(
+          height: 200,
+        )
+      ],
+    );
+  }
+
+  void _handlingOnPageChanged(int page) {
+    setState(() => _currentPage = page);
+  }
+
+  Widget _buildPageIndicator() {
+    Row row = Row(mainAxisAlignment: MainAxisAlignment.center, children: []);
+    for (int i = 0; i < _slides.length; i++) {
+      row.children.add(_buildPageIndicatorItem(i));
+      if (i != _slides.length - 1) {
+        row.children.add(SizedBox(
+          width: 12,
+        ));
+      }
+    }
+    return row;
+  }
+
+  //page indicator circles
+  Widget _buildPageIndicatorItem(int index) {
+    return Container(
+      width: index == _currentPage ? 8 : 5,
+      height: index == _currentPage ? 8 : 5,
+      decoration: BoxDecoration(
+          shape: BoxShape.circle,
+          color: index == _currentPage
+              ? Color.fromRGBO(136, 144, 178, 1)
+              : Color.fromRGBO(206, 209, 223, 1)),
+    );
```

Comment:
Make new variables inside the constants.dart for these colors and use those variable here.